### ISSUE TITLE
FreeBSD: unable to try with npx

### ISSUE THREAD

**Issue by grahamperrin** [sebhildebrandt/systeminformation#850]
**Describe the bug**

From <https://github.com/sebhildebrandt/systeminformation#give-it-a-try-with-npx>: 

`npx systeminformation info`

**To Reproduce**
1. `npx systeminformation info`

**Current Output**

```text
% npx systeminformation info
Need to install the following packages:
  systeminformation@5.21.4
Ok to proceed? (y) y
┌─────────────────────────────────────────────────────────────────────────────────────────┐
│  SYSTEMINFORMATION                                                      Version: 5.21.4 │
└─────────────────────────────────────────────────────────────────────────────────────────┘
/usr/home/grahamperrin/.npm/_npx/e200578d6e6cda24/node_modules/systeminformation/lib/osinfo.js:275
          const logofile = util.getLogoFile(distro);
                                ^

TypeError: util.getLogoFile is not a function
    at /usr/home/grahamperrin/.npm/_npx/e200578d6e6cda24/node_modules/systeminformation/lib/osinfo.js:275:33
    at ChildProcess.exithandler (node:child_process:413:7)
    at ChildProcess.emit (node:events:514:28)
    at maybeClose (node:internal/child_process:1105:16)
    at Socket.<anonymous> (node:internal/child_process:457:11)
    at Socket.emit (node:events:514:28)
    at Pipe.<anonymous> (node:net:337:12)

Node.js v20.5.1
% 
```

**Expected behavior**

2. info :-)

**Environment (please complete the following information):**
 - systeminformation package version: not applicable
 - OS: FreeBSD
 - HP EliteBook 8570p

```text
% date ; uname -aKU
Tue 12 Sep 2023 06:34:51 BST
FreeBSD mowa219-gjp4-8570p-freebsd 15.0-CURRENT FreeBSD 15.0-CURRENT amd64 1500000 #11 main-n265135-07bc20e4740d-dirty: Sat Sep  2 19:40:08 BST 2023     grahamperrin@mowa219-gjp4-8570p-freebsd:/usr/obj/usr/src/amd64.amd64/sys/GENERIC-NODEBUG amd64 1500000 1500000
% 
```

**Additional context**

**Comment by sebhildebrandt** [comment]
@grahamperrin ... thank you for reporting. I need to try this ... will come back with a solution.

**Comment by ovenpasta** [comment]
I've just hit on this one,
The solution is to remove the `util.`, at row 273. the function `getLogoFile `is defined locally.
`const logofile = util.getLogoFile(distro);` becomes `const logofile = getLogoFile(distro);`

**Comment by sebhildebrandt** [comment]
@ovenpasta ... thank you ... sorry, sometimes I am to confused so I have nit seen this simple fix for the issue... Thank you once again. Fixed, version 5.21.11 just released. Can you test It on your side?
